app/views/locations/index.html.erb

```diff
@@ -11,57 +11,102 @@
 
 <%= render partial: "organisations/headings", locals: { main: @scheme.service_name, sub: nil } %>
 
-<%= render SubNavigationComponent.new(items: scheme_items(request.path, @scheme.id, "Locations")) %>
+<% if FeatureToggle.new_locations_table_layout_enabled? %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
```

Comment:
I would personally remove this line because it squishes the table

Reply:
Oh, sorry, I think this is a different page I had in mind. Isn't this page rendering this? 
<img width="1084" alt="image" src="https://user-images.githubusercontent.com/54268893/202414961-3bae987f-0de9-4462-a993-a83fecc8d6da.png">

Reply:
Wait maybe I misread your comment

Reply:
It's fine if that's in designs

Reply:
Sorry if my comment was confusing. The picture above was just to demonstrate the other page (scheme details) which I've _also_ made 2/3 scaling now (in order to match the locations list table) following a decision from Nicky. Happy with the above?

Reply:
Ah sorry I hadn't seen your comment "It's fine if that's in designs" because you made it since I last refreshed the page. Thanks.